### PR TITLE
DeleteCompleteFilesTask

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-716

### Change description ###
DeleteCompleteFilesTask with unit tests
Scheduling config will be added in the next PR

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 22
           

Complexity increasing per file
==============================
- src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTaskTest.java  2
- src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java  5
         

Clones added
============
- src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTaskTest.java  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/bulk-scan-processor/pullRequest?prid=5293418&bid=17328012)

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+        log.info("Finished {} task", TASK_NAME);
+    }
+
+    private void processCompleteFiles(CloudBlobContainer container) {
+        log.info("Started deleting complete files in container {}", container.getName());
+
+        List<Envelope> envelopes = envelopeRepository.findByContainerAndStatus(container.getName(), COMPLETED);
+        int successCnt = 0;
+        int failureCnt = 0;
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java:52]: Let's name with full words here :smile:

**Comment by doncem** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 22
           

Complexity increasing per file
==============================
- src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTaskTest.java  2
- src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java  5
         

Clones added
============
- src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTaskTest.java  5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/HMCTS/bulk-scan-processor/pullRequest?prid=5293418&bid=17328012)

**Comment by apereverzin** [comment]
PR-1124

**Pull request closed without merging**